src/linker/Linker/Driver.cs

```diff
@@ -414,6 +417,25 @@ protected int SetupContext (ILogger customLogger = null)
 
 						continue;
 
+					case "--roots": {
```

Comment:
This should match the docs which say "--root" (I prefer singular FWIW).

Reply:
Not relevant anymore